.github/workflows/ci.yaml

```diff
@@ -9,17 +10,17 @@ jobs:
     container: quay.io/ovirt/buildcontainer:${{ matrix.distro }}
     strategy:
       matrix:
-        distro: [ el8stream, el9stream ]
+        distro: [el8stream, el9stream]
     steps:
-    - name: Prepare env
-      run: mkdir -p ${PWD}/${EXPORT_DIR}/
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-    - name: Check patch
-      run: ./automation/check-patch.sh
-    - name: Upload artifacts
-      uses: ovirt/upload-rpms-action@v1
-      with:
-        directory: ${{ env.EXPORT_DIR }}
-        distro: ${{ matrix.distro }}
+      - name: Prepare env
+        run: mkdir -p ${PWD}/${EXPORT_DIR}/
```

Comment:
I would suggest to move this to the scripts you're calling later. Less mess in this yaml the better...

Reply:
We already sort of do that so this is safe to remove after small changes in the scritps.